lib/widgets/bubbles.dart

```diff
@@ -130,28 +127,29 @@ class DragBubbleState extends State<DragBubble>
                 });
               },
               onPanEnd: invokeDialog(context, homePageState, feature, this),
-              child: Bubble(homePageState, this, feature.value["title"],
-                  feature, color, animationController, bubbleWidth, isSmall)),
+              child: FeatureBubble(
+                  homePageState, this, feature, color, bubbleWidth, isSmall)),
         ),
       ],
     );
   }
 }
 
-class Bubble extends StatelessWidget {
+/// Represents the actual bubble without any logic. Gets it's color and width from parent widget.
+/// [isSmall] decides whether the title of the represented [feature] should be displayed or not.
+/// Has press behavior that opens the dialog just as if it's [DragBubble] was pressed.
+class FeatureBubble extends StatelessWidget {
   final MyHomePageState homePageState;
   final DragBubbleState dragState;
-  final String title;
   final MapEntry<String, dynamic> feature;
   final Color color;
-  final animationController;
   final double bubbleWidth;
   final bool isSmall;
 
-  Bubble(this.homePageState, this.dragState, this.title, this.feature,
-      this.color, this.animationController, this.bubbleWidth, this.isSmall);
+  FeatureBubble(this.homePageState, this.dragState, this.feature, this.color,
+      this.bubbleWidth, this.isSmall);
 
-  composeBubble(context) {
+  List<Widget> _composeBubble(context) {
```

Comment:
whats a compose bubble

Reply:
Compose is the verb. :D A bubble gets composed?

Reply:
You use get everywhere else. GetBubble?

Reply:
Maaan..*once* a better name.. -.- Sure...getBubble.